examples/unix/c99/z_ping.c

```diff
@@ -78,9 +100,9 @@ int main(int argc, char** argv) {
     _z_mutex_unlock(&mutex);
     z_free(results);
     z_free(data);
-    z_undeclare_subscriber(z_move(sub));
-    z_undeclare_publisher(z_move(pub));
-    z_close(z_move(session));
+    z_undeclare_subscriber(z_subscriber_move(&sub));
```

Comment:
don't switch this to z_drop, it's a c99 example (leaving the comment just in case)

Reply:
Done!